Core/tooling/build-final.js

```diff
@@ -7,22 +7,15 @@ const path = {
   bareEntrypointBundle: 'build/bundles/entrypoint-bare-bundle.js',
   finalDev: 'build/final/flowcrypt-android-dev.js',
   finalIos: 'build/final/flowcrypt-ios-prod.js',
-}
-
-// node
-const fixNodeImports = (src) => src
-  .replace(/require\(['"]bn\.js['"]\)/g, 'dereq_bn')
-  .replace(/require\(['"]minimalistic-assert['"]\)/g, 'dereq_minimalistic_assert')
-  .replace(/require\(['"]inherits['"]\)/g, 'dereq_inherits')
-  .replace(/require\(['"]asn1\.js['"]\)/g, 'dereq_asn1')
-  .replace(/require\(['"]encoding-japanese\.js['"]\)/g, 'dereq_encoding_japanese');
+};
 
 // bare
-const bareDepsSrc = fs.readFileSync(path.bareDepsBundle).toString()
-const bareEntrypointSrc = fs.readFileSync(path.bareEntrypointBundle).toString().replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');
+const bareDepsSrc = fs.readFileSync(path.bareDepsBundle).toString();
+const bareEntrypointSrc = fs
+  .readFileSync(path.bareEntrypointBundle).toString()
+  .replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');
```

Comment:
Maybe `  .replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');` could be omitted, it doesn't seem to be doing anything useful, from a casual look. The actual version is set from `CoreHost` somewhere, if I remember. Let's leave it for another PR, this one is already looking good.

Reply:
We set `APP_VERSION` value in `CoreHost` code, so this change is needed for using value provided by `CoreHost`
https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Core/Core.swift#L230

Reply:
Ah, I see.